### ISSUE TITLE
Parse topoguide images links

### ISSUE THREAD

**Issue by arnaud-morvan** [c2corg/v6_forum#49]
In V5 forum topoguide images are inserted in posts using : 
[img=1479113831_1579162882.jpg 821742 inline]un couché de soleil de malade[/img]

Note api image proxy, already permit this :
[img]https://api.demov6.camptocamp.org/images/proxy/815721?size=SI[/img]

**Comment by arnaud-morvan** [comment]
We have to control what is actually migrated for those cases, and avoid data loss.

**Comment by jmarcmod** [comment]
ex.
V6 : http://forum.demov6.camptocamp.org/t/le-jeu-du-sommet/110837/24546?u=jeanmarc
V5 : http://www.camptocamp.org/forums/viewtopic.php?pid=2214098#p2214098

**Comment by arnaud-morvan** [comment]
No data loss, but images are not visible and no links.
This should be handled after Go Live.

**Comment by arnaud-morvan** [comment]
After the last migration which take care of images with title attribute:
```
[img=1479113831_1579162882.jpg 821742 inline]un couché de soleil de malade[/img]
```
see : http://www.camptocamp.org/forums/viewtopic.php?pid=2214098#p2214098

is replaced by 
```
[img]1474642775_516824453.jpg 805275 inline[/img]
```
see: http://datamigration.exoscale.infra.camptocamp.org/t/le-jeu-du-sommet/110838/24546

**Comment by jmarcmod** [comment]
> see

Je n'y ai pas accès, il faut se logger et mes identifiants habituels ne marchent pas.
Tu peux faire une copie d'écran de ce que ça donne ?

**Comment by arnaud-morvan** [comment]
Je ne peut plus, je viens de relancer la migration, tu le verra dans une semaine.

**Comment by jmarcmod** [comment]
@stef74 : cette issue est finalement 6.0 puisqu'elle vient d'être traitée ; on pourra la tester sur la version définitive dans 1 semaine.

**Comment by arnaud-morvan** [comment]
Non, elle n'a pas été traitée, les liens sont légèrement modifiés par la migration, mais ne seront pas affichés correctement dans la V6.0.

**Comment by jmarcmod** [comment]
OK, j'avais mal compris.

@stef74 : désolé pour le dérangement, ça reste bien en 6.1

**Comment by stef74** [comment]
@arnaud-morvan remaining work on this topic ?

**Comment by arnaud-morvan** [comment]
Oui, les balises images, telles que citées plus haut ne sont pas traitées dans la partie forum.
